.env.example

```diff
@@ -1,67 +1,37 @@
-####### API KEYS #######
-#
-# in development environment NONE of the below api keys are required
-# if you leave this file BLANK the app will run via the `ethers` community api key
-# the only side effect may be some latency
-#
+################ API Keys
+# In a development environment, none of the below API keys are required
+# but may be added for improved functionality.
 
-# add your ALCHEMY API KEY
-# - Alchemy IDs are only used as fallbacks (in conjunction with INFURA_IDS)
-# - multiple IDS should be separated by whitespace
-# - in development environment app will run without an ALCHEMY API KEY
-# - you can pick up Alchemy API Keys at https://auth.alchemyapi.io/
-REACT_APP_ETHEREUM_ALCHEMY_IDS=""
+###### Add an Alchemy key
+# You can sign up for an API key at https://auth.alchemyapi.io/
+# and provide that here for improved API speed and latency.
+REACT_APP_ALCHEMY_API_KEY=""
 
-REACT_APP_ARBITRUM_ALCHEMY_IDS=""
-REACT_APP_POLYGON_ALCHEMY_IDS=""
+##### Add an Infura key
+# You can sign up for an API key at https://infura.io/
+# and provide that here for improved API speed and latency.
+REACT_APP_INFURA_API_KEY=""
 
-# add your INFURA ID
-# - Infura IDs are only used as fallbacks (in conjunction with ALCHEMY_IDS)
-# - multiple IDS should be separated by whitespace
-# - you can pick up INFURA_IDS at https://infura.io/
-REACT_APP_INFURA_IDS=""
-
-# add a TESTNET ALCHEMY API KEY
-# - only 1 ID is required, multiple IDS should be separated by whitespace
-REACT_APP_ETHEREUM_TESTNET_ALCHEMY=""
-
-REACT_APP_ARBITRUM_TESTNET_ALCHEMY=""
-
-# if you run your own node you set your https address below
-# - multiple IDS should be separated by whitespace
-# - in Production node URLs must be secure `https://`
-# - in Development node URLs must be insecure `http://`
-REACT_APP_ETHEREUM_SELF_HOSTED_NODE="https://ipAddress:port"
-
-REACT_APP_ARBITRUM_SELF_HOSTED_NODE="https://ipAddress:port"
-REACT_APP_POLYGON_SELF_HOSTED_NODE="https://ipAddress:port"
-REACT_APP_AVALANCHE_SELF_HOSTED_NODE="https://api.avax.network/ext/bc/C/rpc"
-REACT_APP_FANTOM_SELF_HOSTED_NODE="https://rpc.fantom.network https://rpc.ftm.tools"
```

Comment:
let's stick with these. They are the most extendable.

Reply:
Done